docs/tutorials/authentication.md

```diff
@@ -19,6 +19,7 @@ Use the [synapseclient.login][synapseclient.Synapse.login] function
 ```python
 import synapseclient
 syn = synapseclient.login(authToken="authtoken")
+#returns Welcome, First Last! 
```

Comment:
nit: is this something meant to be deleted?

Reply:
I think this is a note for what should be printed back to the user if they have succeeded in authenticating. This shows up in the mkdocs site and I think its pretty helpful.